test/mocha/helpers.js

```diff
@@ -21,9 +21,24 @@ module.exports = api;
 // test hashing function
 api.testHasher = brLedgerNode.consensus._hasher;
 
+let latestBlockHeight = 0;
+api.updateBlockHeight = async ({blockHeight}) => {
+  latestBlockHeight = blockHeight;
+};
+
+api.getLatestBlockSummary = async () => {
```

Comment:
Hmm, this shared state and bespoke API to trip someone up. Any reason this isn't just part of a test or group of tests? Why is it in helpers? Can you just create some `testState` object or something that is shared across multiple tests and access it that way?

I'll make some suggestions.

Reply:
I'll remove the helper functions.

Reply:
Implemented in 18abd3d4a5c879464a57623054c5852d0ed6f00d.